README.md

```diff
@@ -40,6 +40,11 @@ If you want to get started quickly and learn how to build on top of Cosmos SDK,
 
 For more information, see the [Cosmos SDK Documentation](https://docs.cosmos.network).
 
+## Modules
+
+The Cosmos SDK maintains a set of modules that can be included in your blockchain application.  For more information
```

Comment:
here we say we maintain a set of modules that _can_ be included, but in the x/readme we say essentials are _required_. should we make that distinction here?